public/lib/helpers/timeline_helper.js

```diff
@@ -122,6 +122,22 @@ define(function (require) {
       return sortObj;
     };
 
+    /**
+     * Creates an Elasticsearch sort object to sort in chronological order on end field
+     *
+     * @param params group configuraton parameters
+     * @returns Elasticsearch sort object
+     */
+    TimelineHelper.prototype.getSortOnEndFieldObject = function (params) {
+      const sortObj = {};
+      if (params.endFieldSequence) {
+        sortObj[params.endFieldSequence.join('.')] = { order: 'asc' };
```

Comment:
It is better to pass the order (`asc`) as a parameter to this method. In this way, you make it easier to read the code in the https://github.com/sirensolutions/kibi_timeline_vis/blob/master/public/kibi_timeline.js